assets/package.json

```diff
@@ -1,19 +1,16 @@
 {
   "name": "@massds/mayflower-tokens",
-  "version": "1.0.1",
+  "version": "1.3.0",
   "description": "Mayflower design tokens — it includes color token SCSS variables, fonts, icons and other imagery",
-  "style": "colors/_mayflower-color-tokens.scss",
+  "style": "dist/scss/_colors.scss",
```

Comment:
Does it make sense to have a default export here?  I think if we do, it should be an export of all of the variables, not just colors.  I guess we could always switch that later though, as long as we're only adding variables to the default export, not removing them.

Reply:
Do you mean an index.scss under dist/scss that imports _colors.scss and _fonts.scss?

Reply:
We originally talked about only including colors, fonts, icons into the mayflower-tokens package, are there other things you think make sense to include?